charts/rode-collector-harbor/values.yaml

```diff
@@ -81,7 +81,25 @@ debug: false
 
 rode:
   host: localhost:50051
-  insecure: false
+
+  ## disableTransportSecurity will disable GRPC transport security when set to `true`
+  ##
+  disableTransportSecurity: false
+
+  ## Rode authentication settings
+  ## Use rode.auth.oidc.enabled to enable OIDC auth
+  ## Use rode.auth.basic.enabled to enable basic auth
+  ##
+  auth:
+    oidc:
```

Comment:
I think we'll want to be able to configure scopes and `tlsInsecureSkipVerify` here too